src/click/_termui_impl.py

```diff
@@ -377,16 +378,16 @@ def pager(generator: t.Iterable[str], color: t.Optional[bool] = None) -> None:
     if os.environ.get("TERM") in ("dumb", "emacs"):
         return _nullpager(stdout, generator, color)
     if WIN or sys.platform.startswith("os2"):
-        return _tempfilepager(generator, "more <", color)
-    if hasattr(os, "system") and os.system("(less) 2>/dev/null") == 0:
+        return _tempfilepager(generator, "more", color, pipe=True)
```

Comment:
I noticed `_pipepager` exists now. Should we not use that instead here and remove the `pipe` argument I introduced?

Reply:
If that works as well, I'm usually in favor of avoiding new arguments.

Reply:
I'll try to test this on Windows and get this merged then.